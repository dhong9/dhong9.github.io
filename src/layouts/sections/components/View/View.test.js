import renderer from "react-test-renderer";
import View from "layouts/sections/components/View";

// Define Mocks
jest.mock("components/MKBox", () => {
    const { forwardRef } = jest.requireActual("react");
    return {
        __esModule: true,
        default: forwardRef(() => <div>Mock Box</div>)
    };
});
jest.mock("components/MKTypography", () => {
    const { forwardRef } = jest.requireActual("react");
    return {
        __esModule: true,
        default: forwardRef(() => <p>Mock Typography</p>)
    };
});

describe("View", () => {
    it("renders", () => {
        const component = renderer.create(<View title="Test View" code="" height="100%" >Test View</View>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});