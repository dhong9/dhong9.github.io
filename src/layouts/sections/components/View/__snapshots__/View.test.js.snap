// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View renders 1`] = `
<div>
  Mock Box
</div>
`;
